pkg/config/config.go

```diff
@@ -1056,6 +1062,13 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 		return err
 	}
 
+	if c.Timezone != "" && !strings.EqualFold(c.Timezone, "local") {
```

Comment:
I think this could be simplified to just the following:
```suggestion
	if c.Timezone != "local" {
```

Why? There is no UTF-8 or Unicode in the TZ identifier to speak of.

Reply:
I believe we shouldn't solely check for 'local' because if the time zone is empty (' '), it enters that condition and executes `LoadLocation`, which returns UTC. Our hypothesis relies on the assumption that CRI-O will not take any action if the time zone is an empty string.

Reply:
Ahh. Makes sense. Good point. My focus was to remove the comparison function useful for Unicode. Which will not be the case here, I believe.